src/site/content/en/blog/web-animations/index.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Web Animations API improvements in Chromium 84
+subhead: Orchestrating animations with promises, performance improvements with replaceable animations, smoother animations with composite modes, and more.
+authors:
+  - una
+  - kevinellis
+date: 2020-05-27
+hero: hero.png
```

Comment:
I can't remember if we checked for this before, but can you try **lightly** optimizing this hero image with Squoosh? We have an image CDN so you shouldn't over-optimize it (or else it'll show up grainy for readers). The optimization is to keep the size of the repository down.